canonical/ubuntu.com
# Exclude the lang-kr from the blog listing

## Done

- Exclude the lang-kr from the blog listing

## QA

- Visit the /blog listing and see there are no Korean posts
- Visit [production](https://ubuntu.com/blog) and see that there is
